service/manager/manager.go

```diff
@@ -0,0 +1,24 @@
+package manager
+
+import (
+	"github.com/mesg-foundation/core/service"
+)
+
+// Manager is responsible for managing Docker Containers of MESG services.
+// it can be implemented for any container orchestration tool.
+type Manager interface {
```

Comment:
I feel `Manager` is too generic, what do you think about `Runner`

Reply:
I don't mind but maybe we can come up with a much better name than these too. let's think about this, we can always change later